### ISSUE TITLE
Support for scraping market places

### ISSUE THREAD

**Issue by okandas** [kevinzg/facebook-scraper#567]
Could we add supoort for scraping market places ?

**Comment by neon-ninja** [comment]
It's already supported, try the `get_shop` function.

**Comment by okandas** [comment]
thank you could we add it to the docs

**Comment by neon-ninja** [comment]
No, a shop is similar to this URL: https://www.facebook.com/SwaziNZ/shop/?_rdr

**Comment by okandas** [comment]
okay, so we can work on it to work for market places

**Comment by okandas** [comment]
@neon-ninja How are you ? Is it possible that we can work on this .. I would like to make a PR for this

**Comment by neon-ninja** [comment]
Of course, go ahead, pull requests are welcome 👍

**Comment by backpapp** [comment]
> @neon-ninja How are you ? Is it possible that we can work on this .. I would like to make a PR for this

How did you go with this ?

**Comment by neon-ninja** [comment]
@okandas still hasn't submitted their PR

**Comment by okandas** [comment]
sorry guy bit swamped

**Comment by rafaelribeiroo** [comment]
Any updates?
